### ISSUE TITLE
migration in 5.1

### ISSUE THREAD

**Issue by krtschmr** [doorkeeper-gem/doorkeeper#968]
```
StandardError: An error has occurred, all later migrations canceled:

Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:

  class CreateDoorkeeperTables < ActiveRecord::Migration[4.2]

```

**Comment by 46bit** [comment]
I fixed this for me by editing `db/migrate/..._create_doorkeeper_tables.rb` to have its first line be: 

```ruby
class CreateDoorkeeperTables < ActiveRecord::Migration[5.1]
```

The code generation probably wants to detect the Rails version and put it in?

**Comment by nbulaj** [comment]
@maclover7 I think we need to fix it for Rails >= 5.0

**Comment by DmitryBash** [comment]
@nbulaj I correctly understood. You want to gsub in 'db/migrate/create_doorkeeper_tables.rb' version of migration?

**Comment by nbulaj** [comment]
@DmitryBash yep, it is a possible solution - [gsub a migration](http://guides.rubyonrails.org/generators.html#gsub-file) from the generator (install action) by adding the version for later Rails.

**Comment by DmitryBash** [comment]
@nbulaj https://github.com/doorkeeper-gem/doorkeeper/pull/977

**Comment by akinrt** [comment]
@DmitryBash hi, thank you for your update for supporting rails 5 :)
i found several way to get the version from the code in rails. 

```ruby
ActiveRecord::Migration.current_version
=> 5.1

ActiveRecord::VERSION::STRING.to_f
=> 5.1

Rails.version.to_f
=> 5.1
```

https://github.com/rails/rails/blob/1729d3b4b2ea96becbb473c982a78d2cf957ce6d/activerecord/lib/active_record/migration.rb#L536-L538

https://github.com/rails/rails/blob/v5.0.0.beta1/activerecord%2Flib%2Factive_record%2Fmigration.rb#L502-L504
